examples/stable-diffusion/text_to_image_generation.py

```diff
@@ -202,25 +202,31 @@ def main():
     parser.add_argument(
         "--ldm3d", action="store_true", help="Use LDM3D to generate an image and a depth map from a given text prompt."
     )
+    parser.add_argument(
+        "--throughput_warmup_steps",
+        type=int,
+        default=None,
```

Comment:
If this is not given, I see that it uses 3 by default in denoising step. Then it's better to give default value 3 here?

Reply:
It may be better to keep default to be given in the pipeline itself, so different pipelines may have specific warmup defaults